packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -132,6 +133,17 @@ export const ProgressIndicatorItem = ({
 										'&:hover': {
 											backgroundColor: hoverColorMap[background],
 											...packs.underline,
+
+											// Extra highlighted space on hover for visual balance (not when focused which avoids hover styles covering the focus ring)
+											':not(:focus)::before': {
```

Comment:
We use `:focus-visible` so should this match, or doesn't it matter? If I enable `:focus-visible` in dev tools, I get this 
<img width="694" alt="image" src="https://github.com/agriculturegovau/agds-next/assets/853552/4ac47afa-0620-47ed-bd82-a2e2007db42a">

Reply:
When I use keyboard tabbing to send focus it looks normal, even when I hover over it with my mouse

![image](https://github.com/agriculturegovau/agds-next/assets/10610368/ef062845-dbbd-4d98-af71-e1816ec99935)

Reply:
`':not(:focus-visible)::before'` works better. I'll update.

Reply:
@Dan503 yep, works fine until it's clicked, then the extra grey is missing.